src/modules/Tab/Tab.js

```diff
@@ -37,6 +39,9 @@ class Tab extends Component {
     /** Shorthand props for the Menu. */
     menu: PropTypes.object,
 
+    /** Shorthand props for the Grid. */
+    grid: PropTypes.object,
```

Comment:
@levithomason I went ahead and made the omit/rest change.  Let me know if you decide you want to use a different approach.